src/Stratis.Bitcoin.Features.Wallet/WalletFeePolicy.cs

```diff
@@ -33,17 +36,35 @@ public class WalletFeePolicy : IWalletFeePolicy
         /// </summary>
         private readonly FeeRate minRelayTxFee;
 
+        /// <summary>Header chain.</summary>
+        private readonly ConcurrentChain chainState;
+
+        /// <summary>Repository containing blocks.</summary>
+        private readonly IBlockRepository blockRepository;
```

Comment:
Dont use `IBlockRepository` use IBlockStore instead.
Also I think wallet does not have a dependency on IBlockStore being present so you need to allow it to be null and I guess use `minRelayTxFee` if store is null.